lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -90,16 +90,21 @@ function gutenberg_resolve_assets_override() {
 	$block_registry = WP_Block_Type_Registry::get_instance();
 
 	foreach ( $block_registry->get_all_registered() as $block_type ) {
-		$style_handles = array_merge(
-			$style_handles,
-			$block_type->style_handles,
-			$block_type->editor_style_handles
-		);
-
-		$script_handles = array_merge(
-			$script_handles,
-			$block_type->script_handles
-		);
+		// In older WordPress versions, like 6.0, these properties are not defined.
+		if ( isset( $block_type->style_handles ) && isset( $block_type->editor_style_handles ) ) {
```

Comment:
Maybe it'd be better to use `is_array` instead of `isset`?

Reply:
Actually, I think `isset` could cause issues. I don't know if it'd be possible for these methods to return anything truthy that's not an array, but it'd return `true` for say, a string, and `array_merge` would break with a string.